diff --git a/MODULE.bazel b/MODULE.bazel
new file mode 100644
index 0000000..d845d14
--- /dev/null
+++ b/MODULE.bazel
@@ -0,0 +1,12 @@
+###############################################################################
+# Bazel now uses Bzlmod by default to manage external dependencies.
+# Please consider migrating your external dependencies from WORKSPACE to MODULE.bazel.
+#
+# For more details, please check https://github.com/bazelbuild/bazel/issues/18958
+###############################################################################
+
+module(name="com_google_file_based_test_driver")
+bazel_dep(name = "abseil-cpp", version = "20240116.2", repo_name = "com_google_absl")
+bazel_dep(name = "googletest", version = "1.14.0.bcr.1", repo_name = "com_google_googletest")
+bazel_dep(name = "re2", version = "2024-06-01", repo_name = "com_googlesource_code_re2")
+bazel_dep(name = "platforms", version = "0.0.10")
diff --git a/WORKSPACE.bazel b/WORKSPACE.bazel
new file mode 100644
index 0000000..ce1dff0
--- /dev/null
+++ b/WORKSPACE.bazel
@@ -0,0 +1 @@
+workspace(name="com_google_file_based_test_driver")
diff --git a/file_based_test_driver/base/BUILD b/file_based_test_driver/base/BUILD
index 9279d64..ae98d7c 100644
--- a/file_based_test_driver/base/BUILD
+++ b/file_based_test_driver/base/BUILD
@@ -259,6 +259,10 @@ cc_library(
         "@com_google_absl//absl/strings:str_format",
         "@com_googlesource_code_re2//:re2",
     ],
+    copts = select({
+        "@platforms//os:windows" : [],
+        "//conditions:default" : ["-Wno-char-subscripts"],
+    })
 )
 
 cc_library(
diff --git a/file_based_test_driver/base/source_location.h b/file_based_test_driver/base/source_location.h
index d67df2f..ab0f67c 100644
--- a/file_based_test_driver/base/source_location.h
+++ b/file_based_test_driver/base/source_location.h
@@ -133,7 +133,6 @@ class SourceLocation {
   std::uint_least32_t line_;
   std::uint_least32_t unused_column_ = 0;
   const char* file_name_;
-  const char* unused_function_name_ = nullptr;
 };
 
 }  // namespace file_based_test_driver_base
